Benchmark/Coreutils.hs

```diff
@@ -0,0 +1,75 @@
+module Main where
+
+import Streamly.Coreutils
+import qualified Streamly.Coreutils.Uniq as U
+
+import qualified Streamly.Prelude as S
+import qualified Streamly.Memory.Array as A
+
+import qualified Streamly.Data.Unicode.Stream as Un
+import qualified Streamly.Internal.FileSystem.File as File
+import Streamly.Internal.Data.Stream.Serial (SerialT(..))
+import Streamly.Internal.Data.Stream.StreamK.Type (IsStream)
+
+import Data.Word (Word8)
+import Control.Monad.IO.Class (MonadIO)
+import System.IO (
+         Handle
+       , stdout
+       , hPutStr
+       , IOMode(..)
+       , openFile)
+
+import Gauge (defaultMain)
+import Gauge.Benchmark (bench, bgroup, nfAppIO, nfIO)
+import Control.Monad.Catch (MonadThrow, MonadCatch)
+
+
+srcFP :: FilePath
+srcFP = "/home/shruti/sorted.txt"
```

Comment:
You might want to remove your name?

Reply:
Should we not push this file at all?

Reply:
I will write property based tests and those can be pushed.

Reply:
You should probably use some function to get the current directory.

Reply:
Okay. :+1: good idea